src/tests/topp/ProteomicsLFQ_5_out.mzTab

```diff
@@ -108,164 +108,164 @@ PRT	tr|A9G8G1|A9G8G1_SORC5	Putative uncharacterized protein OS=Sorangium cellulo
-PEP	KQTALVELLK	P02769|ALBU_BOVIN	1	null	null	null	0.0	null	null	null	null	0.0	null	null	2475.227593453340887	null	2	571.860808982021013	ms_run[5]:spectrum=3459	null	null	null	5.3558568e07	null	null	null	null	null	KQTALVELLK	4314556777634898838	2.62096624298549e-05	0	571.860808982021013	2475.227593453340887	null	null	null	null
-PEP	RHPEYAVSVLLR	P02769|ALBU_BOVIN	1	null	null	null	0.0	null	null	null	null	0.0	null	null	2488.240298282809817	null	3	480.608772593904405	ms_run[5]:spectrum=3369	null	null	null	1.2556631e07	null	null	null	null	null	RHPEYAVSVLLR	5251706238937085209	9.077820128403891e-06	0	480.608772593904405	2488.240298282809817	null	null	null	null
+PEH	sequence	accession	unique	database	database_version	search_engine	best_search_engine_score[1]	search_engine_score[1]_ms_run[1]	search_engine_score[1]_ms_run[2]	search_engine_score[1]_ms_run[3]	search_engine_score[1]_ms_run[4]	search_engine_score[1]_ms_run[5]	search_engine_score[1]_ms_run[6]	modifications	retention_time	retention_time_window	charge	mass_to_charge	spectra_ref	peptide_abundance_study_variable[1]	peptide_abundance_stdev_study_variable[1]	peptide_abundance_std_error_study_variable[1]	peptide_abundance_study_variable[2]	peptide_abundance_stdev_study_variable[2]	peptide_abundance_std_error_study_variable[2]	peptide_abundance_study_variable[3]	peptide_abundance_stdev_study_variable[3]	peptide_abundance_std_error_study_variable[3]	opt_global_cv_MS:1000889_peptidoform_sequence	opt_global_feature_id	opt_global_FFId_category	opt_global_OMSSA_score	opt_global_cv_MS:1002217_decoy_peptide	opt_global_mass_to_charge_study_variable[2]	opt_global_retention_time_study_variable[2]	opt_global_mass_to_charge_study_variable[3]	opt_global_retention_time_study_variable[3]	opt_global_mass_to_charge_study_variable[1]	opt_global_retention_time_study_variable[1]
+PEP	DGDIEAEISR	tr|A9EY18|A9EY18_SORC5	1	null	null	null	0.0	null	0.0	null	null	null	null	null	1523.370634117893133	null	3	368.843772909670918	ms_run[2]:spectrum=2311	null	null	null	2788.036376953125	null	null	null	null	null	DGDIEAEISR	2935923263525422257	internal	8.461750035304259e-04	0	368.843772909670918	1523.370634117893133	null	null	null	null
+PEP	SHCIAEVEK	P02769|ALBU_BOVIN	1	null	null	null	0.0	null	null	0.0	null	null	null	3-UNIMOD:4	1533.479042701062554	null	3	358.174576486337685	ms_run[3]:spectrum=2376	null	null	null	null	null	null	4.618461875e05	null	null	SHC(Carbamidomethyl)IAEVEK	14086301720075657476	internal	6.76118991622199e-03	0	null	null	358.174576486337685	1533.479042701062554	null	null
+PEP	EVEAICHSK	P02769|ALBU_BOVIN_rev	1	null	null	null	0.0	0.0	null	null	null	null	null	6-UNIMOD:4	1558.221639541818604	null	3	358.174576486337628	ms_run[1]:spectrum=2458	1.35815075e06	null	null	null	null	null	null	null	null	EVEAIC(Carbamidomethyl)HSK	5634796270667792442	internal	6.24514018064888e-04	1	null	null	null	null	358.174576486337628	1558.221639541818604
```

Comment:
why was internal missing in the old version?
I see it in the PSM section though

Reply:
Just looking at the diffs

Reply:
Do we need to change anything here before this can get merged?